project-base/config/packages/framework_extra.yml

```diff
@@ -0,0 +1,3 @@
+sensio_framework_extra:
+    router:
+        annotations: false
```

Comment:
Can you please explain this to me? 
why it's needed to add it and for what the bundle is used

Reply:
Becose symfony by default use `sensio_framework` routing and we don't want this. You can look at this: https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/576, or this: https://github.com/symfony/recipes/pull/430

Reply:
I see... and where is the bundle used? 
From https://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/index.html I don't see any relevant usages in our controllers

Reply:
I see we removed the dependency in a different comment :D

Reply:
This is not using in our code, but framework-extra-bundle is  loaded and register by another packages (for example `trikoder/oauth2-bundle`). Therefore I need to disable the default routing for sensio.